Source/WarpX.H

```diff
@@ -1450,7 +1450,7 @@ private:
     amrex::Real v_particle_pml;
 
     amrex::Real moving_window_x = std::numeric_limits<amrex::Real>::max();
-    amrex::Real current_injection_position = 0;
+    amrex::Vector<amrex::Real> current_injection_position;
```

Comment:
Could we store this as a member variable of `PhysicalParticleContainer`, instead of storing it as an array in the WarpX class?

Reply:
I think we would have to store it as a member variable of `WarpXParticleContainer` because we would access it from objects like `pc` above, defined as `WarpXParticleContainer& pc = mypc->GetParticleContainer(i);`. Would you want to have it in `WarpXParticleContainer`?

Reply:
Discussed offline: let's add `current_injection_position` as a member variable of the class `WarpXParticleContainer`.

Reply:
Done in [15bad72](https://github.com/ECP-WarpX/WarpX/pull/3958/commits/15bad72491d41c15c6ed251a5278727b9c7007a5).